features/aanhef.feature

```diff
@@ -699,10 +699,40 @@ Rule: Wanneer de geslachtsnaam van de persoon leeg of onbekend is en de naam van
 
     Voorbeelden:
       | naamgebruik | aanhef                  | leveren naam |
-      | E           |                         | GEEN         |
       | P           | Geachte mevrouw De Boer | WEL          |
-      | V           |                         | GEEN         |
-      | N           |                         | GEEN         |
+
+  Abstract Scenario: naam van de persoon is onbekend bij aanduiding naamgebruik "<naamgebruik>"
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                        | waarde    |
+    | burgerservicenummer         | 999992934 |
+    | geslachtsaanduiding (04.10) | V         |
+    En de persoon heeft de volgende 'naam' gegevens
+    | naam                                 | waarde        |
+    | voornamen (02.10)                    |               |
+    | adellijke titel of predicaat (02.20) |               |
+    | voorvoegsel (02.30)                  |               |
+    | geslachtsnaam (02.40)                |               |
```

Comment:
doe je hier expres een lege geslachtsnaam in plaats van een punt (.) als standaardwaarde?

Reply:
Nee, dit is zoals het al stond, maar is nu aangepast